website/content/docs/job-specification/network.mdx

```diff
@@ -294,6 +304,24 @@ network {
 
 The Nomad client will build the correct [capabilities arguments](https://github.com/containernetworking/cni/blob/v0.8.0/CONVENTIONS.md#well-known-capabilities) for the portmap plugin based on the defined port blocks.
 
+### CNI Args
+
+The following example specifies CNI args for the custom CNI plugin specified above.
+
+```hcl
+network {
+  mode = "cni/mynet"
+  port "http" {
+    to = 8080
+  }
+  cni {
+    args = {
+     "nomad.region" : "${node.region}"
```

Comment:
seeing the `.` here in the map key made me realize that people could include a `;` in either key or value, and accidentally cause themselves mysterious issues, since the CNI_ARGS env var ends up as a `;`-separated string.

I think we should prevent that from happening (add just a touch more validation).

Reply:
but side-note, seeing this example here with interpolation really excites the imagination!